monitor/getBalances.js

```diff
@@ -103,6 +105,18 @@ async function main(bridgeMode) {
       logger.debug('Methods for half duplex token are not present')
     }
 
+    try {
+      logger.debug('calling foreignBridge.methods.chaiToken')
+      const chaiToken = await foreignBridge.methods.chaiToken().call()
+      if (chaiToken !== ZERO_ADDRESS) {
+        displayChaiToken = true
+        logger.debug('calling foreignBridge.methods.investedAmountInDai')
+        investedAmountInDai = await foreignBridge.methods.investedAmountInDai().call()
+      }
+    } catch (e) {
+      logger.debug('Methods for chai token are not present')
```

Comment:
If you are going to use `isChaiTokenEnabled` as suggested above you need also report something like "support of Chai token is disabled"

Reply:
Added logging message for disabled chai.